### PR TITLE
DEVELOPER-4031 - New webpage to verify program membership

### PR DESCRIPTION
**Pull request by KamiQuasi**
https://issues.jboss.org/browse/DEVELOPER-4031

Existing member should receive an affirmative upon press of enter or blur of the input field. Non-member should receive negative and registration link upon press of enter or blur of the field.

Content must be added and directed to /verify-member

### CODE REVIEW

**Comment by KamiQuasi** [comment]
retest this please

**Comment by KamiQuasi** [comment]
retest this please

File: javascripts/verify-member.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+var verifyMemberApp = angular.module('member-verify', []);
+
+verifyMemberApp.controller('VerifyCtrl', ['$scope', 'validateMember', function VerifyCtrl($scope, validateMember) {
+    var eChk = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```
Comment (LightGuard) [inline javascripts/verify-member.js]: Eewww... could you break this down in comments, please? I don't think anyone, including future us, wants to try and figure out what this is doing. It almost looks like this is an email checking, but seems like there's too many '@'.

File: stylesheets/_verify.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    font-size: 1.4em;
+    font-weight: 600;
+    display: inline;
+}
+
+.verify-member input {
+    padding: .5em;
+    margin-left: 1em;
+    width: 250px;
+}
```
Comment (LightGuard) [inline stylesheets/_verify.scss]: If we're using scss, wouldn't this be easier to read using the nesting syntax?

File: stylesheets/_verify.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+.isMember {
+    font-size: 5em;
+    color: green;
+    font-weight: 600;
+    margin-bottom: 0;
+}
+
+.verify-member h5 {
```
Comment (LightGuard) [inline stylesheets/_verify.scss]: This should probably be grouped with the other .verify-member section.

File: javascripts/verify-member.js (hunk 1 above)
Comment (LightGuard) [inline javascripts/verify-member.js]: Please explain what the regex is doing in comments.

**Comment by KamiQuasi** [comment]
retest this please

**Comment by LightGuard** [comment]
retest this please

**Comment by robpblake** [comment]
retest this please

**Comment by KamiQuasi** [comment]
@LightGuard and @adelasofia the failing tests don't appear to have any relation to the changes as near as I can tell. The sooner we can get this merged to test the service call and mobile view on the live environment the better.

**Comment by LightGuard** [comment]
Can we test in the PR environment?

**Comment by LightGuard** [comment]
We're still waiting for the Drupal changes to mark this as complete.

**Pull request merged**